content/docs/install/helm-install/index.md

```diff
@@ -4,65 +4,126 @@ description: Install Keptn on a single cluster using the Helm chart
-  [Continuous Delivery](../../concepts/delivery/)
-  on top of Quality Gates and Automated Operations..
-  * Note that you must install either the
-  [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
-  or [Istio](https://istio.io) on the Kubernetes cluster(s) where the Execution Plane is installed.
+
+  The Control Plane can implement the [Quality Gates](../../concepts/quality_gates/)
+  and [Automated Operations](../../concepts/automated_operations/) features
+  but does not run microservices that integrate other tools with Keptn.
+
+* The **Execution Plane** refers to the microservices that integrate other tools with Keptn;
+  see [Keptn and other tools](../../concepts/keptn-tools).
+  For example, the JMeter test tool 
+  and the [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
+  execute on the Execution Plane.
 
 See [Architecture](../../concepts/architecture) for more information
 about the Control Plane and the Execution Plane.
 
-Keptn can be installed on an existing Kubernetes cluster that hosts other software
+You have the following installation options:
+
+* Both the Keptn Control Plane and the Execution Plane
+can be installed on an existing Kubernetes cluster that hosts other software
 or on a dedicated Keptn Kubernetes cluster.
-The Control Plane and Execution Plane can also be installed on separate Kubernetes clusters;
+* The Control Plane and Execution Plane can also be installed on separate Kubernetes clusters;
 see [Multi-cluster setup](../multi-cluster) for instructions.
 * In a multi-cluster setup, one Keptn Control Plane can support multiple Execution Planes
 that are installed on different clusters.
 
-## Control Plane installation options
-
-You must specify the [access option](../access) when you install
-the Keptn Control Plane.
-
-* Install Keptn control-plane with Continuous Delivery support and exposed on a [LoadBalancer](../access/#option-1-expose-keptn-via-a-loadbalancer):
+## Install Control Plane
+
+To install the Control Plane, you must do the following:
+
+* Define the Keptn chart repository
+* Install Keptn into the `keptn` namespace on the Kubernetes cluster
+* Expose the [API gateway](../../concepts/architecture/#api-gateway-nginx) service
+  that controls how Keptn communicates with the internet.
+  See [Choose access option](../access) for details about all the options
+  that are available and how to install and use them.
+  You can also do this after installation.
+* You may also want to modify the Keptn configuration options.
+  This is discussed more below.
+
+### Simple Keptn installation
+
+  The following commands provide a basic Keptn installation,
+  Because the install command does not expose the API gateway service,
+  we use the **kubectl** command to implement port forwarding,
+  which implements the Kubernetes [LoadBalancer](../access/#option-1-expose-keptn-via-a-loadbalancer):
```

Comment:
```suggestion
  We use `kubectl` to forward port 8080 from our local machine to port 8080 on Keptn's API Gateway service in the cluster:
```

Reply:
Done.  Can't make Keptn possessive without voiding the trademark alas, so I removed the 's.